File: einheit03/uebung/e3a3/e3a3.m
disp('*** aufgabe 3');
clear

a = 5;
x = linspace (-3,3,40);
f = 1./(x.^2+a);

%alternative waere fplot zu nutzen. Dann braucht man kein linspace.

figure
plot (x,f,'Linewidth',2);
grid on; box on;
title('$f(x) = \frac{1}{x^2+a}$','interpreter','latex')

File: einheit03/uebung/e3a3/e3a3.py
# -*- coding: utf-8 -*-
"""
Created on Thu Sep 26 13:49:11 2013

@author: jschulz1
"""

import numpy as np  # NumPy (multidimensional arrays, linear algebra, ...)
import scipy as sp  # SciPy (signal and image processing library)
import matplotlib as mpl         # Matplotlib (2D/3D plotting library)
import matplotlib.pyplot as plt  # Matplotlib's pyplot: MATLAB-like syntax
from pylab import *              # Matplotlib's pylab interface

a = 5
x = linspace (-3,3,40)
f = 1./(x**2+a)

figure()
plot (x,f,linewidth=2)
grid('on')
box('on')
title('$f(x) =\\frac{1}{x^2+a}$')
show()
